### ISSUE TITLE
About the limited gpus for few-shot detection?

### ISSUE THREAD

**Issue by xiaofeng-c** [open-mmlab/mmfewshot#28]
How many gpus will be used for the few-shot detection,whether it support 1 gpu?

**Comment by linyq17** [comment]
Sure, but some hyper-parameters, such as learning rate, need to be tuned for your setting. More details about basic usages can be found in the documents.

**Comment by xiaofeng-c** [comment]
Ok,i get it,thank you.
